### PR TITLE
Fixed #28667 -- Clarified how to override list of forms fields for custom UserAdmin with custom user model.

### PR DESCRIPTION
**Pull request by Swat009**
https://code.djangoproject.com/ticket/28667

### CODE REVIEW

File: docs/topics/auth/customizing.txt [hunk 1]
```diff
@@ -880,6 +880,36 @@ override any of the definitions that refer to fields on
 ``django.contrib.auth.models.AbstractUser`` that aren't on your
 custom user class.
 
+It is important to note that if you are using a custom ``ModalAdmin`` which is a
+subclass of ``django.contrib.auth.admin.UserAdmin`` your custom fields need to
+be added to ``fieldsets`` for ``UserChangeForm`` and ``add_fieldsets`` for
+``UserCreactionForm`` of ``UserAdmin`` instead of overriding the fields
+in the ``Meta`` of your custom form as shown below::
```
Comment (felixxm) [inline docs/topics/auth/customizing.txt]: I would simplify this and refer to a full example which contains detailed explanation, e.g.
```diff
diff --git a/docs/topics/auth/customizing.txt b/docs/topics/auth/customizing.txt
index 007431ba33..c3cbf21aa3 100644
--- a/docs/topics/auth/customizing.txt
+++ b/docs/topics/auth/customizing.txt
@@ -880,6 +880,14 @@ override any of the definitions that refer to fields on
 ``django.contrib.auth.models.AbstractUser`` that aren't on your
 custom user class.
 
+.. note::
+
+    If you're using a custom ``ModalAdmin`` which is a subclass of
+    ``django.contrib.auth.admin.UserAdmin``, then your custom fields need to be
+    added to ``fieldsets`` (for fields to be used in editing users) and to
+    ``add_fieldsets`` (for fields to be used when creating a user). See
+    a custom :ref:`UserAdmin class <custom-users-admin-full-example>` example.
+
 Custom users and permissions
 ----------------------------
 
@@ -974,6 +982,8 @@ If your project uses proxy models, you must either modify the proxy to extend
 the user model that's in use in your project, or merge your proxy's behavior
 into your :class:`~django.contrib.auth.models.User` subclass.
 
+.. _custom-users-admin-full-example:
+
 A full example
 --------------
 
```

**Comment by felixxm** [comment]
buildbot

File: docs/topics/auth/customizing.txt [hunk 2]
```diff
@@ -880,6 +880,36 @@ override any of the definitions that refer to fields on
 ``django.contrib.auth.models.AbstractUser`` that aren't on your
 custom user class.
 
+It is important to note that if you are using a custom ``ModalAdmin`` which is a
```
[the remark this answers is not in the document]
Comment (chason) [reply docs/topics/auth/customizing.txt]: Should ``ModalAdmin`` be ``ModelAdmin``?

**Comment by Swat009** [comment]
Made the suggested changes.

**Comment by felixxm** [comment]
@Swat009 Thanks !:+1: I rebased from master, squashed commits and restored a short example (I changed my mind, it makes sense to keep it).

**Pull request merged**